WebRoot/js/zimbraAdmin/globalconfig/model/ZaGlobalConfig.js

```diff
@@ -21,19 +21,21 @@
  * ***** END LICENSE BLOCK *****
  */
 
-ZaGlobalConfig = function() {
-	ZaItem.call(this,"ZaGlobalConfig");
-	this.attrs = new Object();
-	this.type = ZaItem.GLOBAL_CONFIG;
-//	this.attrsInternal = new Object();	
-	this.load();
+ZaGlobalConfig = function () {
+    ZaItem.call(this, "ZaGlobalConfig");
+    this.attrs = new Object();
+    this.type = ZaItem.GLOBAL_CONFIG;
+    //this.attrsInternal = new Object();	
+    this.load();
+    this._init();
```

Comment:
Does `this` always have `_init` function in any case? I'm worried about an exception due to undefined function.